src/generateSdk/sdkGeneratorHandler.ts

```diff
@@ -25,7 +25,7 @@ export async function generateSdk(
 
     if (plugins) {
         pluginsImported = plugins?.map(async (plugin) => {
-            return await import(plugin).catch((err: any) => {
+            return await import(plugin).catch(() => {
```

Comment:
Here do not remove `err`, display the error for the verbose output.

You can do that with `debugLogger.debug()`.